src/coreir-c/values-c.cpp

```diff
@@ -67,6 +67,12 @@ extern "C" {
     return rcast<COREValue*>(ga);
   }
 
+  COREValue* COREValueModule(COREContext* cc, COREModule* mod) {
+    Context* c = rcast<Context*>(cc);
+    Value* ga = Const::make(c, (Module *)mod);
```

Comment:
Is there a reason that this is not using an rcast?

Reply:
Too much C programming lately heh, good catch